.github/workflows/install-dependencies.sh

```diff
@@ -19,8 +19,8 @@ set -euo pipefail
 sudo apt-get -y install git kubectl maven default-jdk python3-pip python3-venv
 
 # install go
-curl -O https://storage.googleapis.com/golang/go1.12.9.linux-amd64.tar.gz
-tar -xvf go1.12.9.linux-amd64.tar.gz
+curl -O https://storage.googleapis.com/golang/go1.17.5.linux-amd64.tar.gz
+tar -xvf go1.17.5.linux-amd64.tar.gz
```

Comment:
Now that we're using Go 1.17, do we need to change `go get -u github.com/google/addlicense` to [`go install github.com/google/addlicense@latest`](https://go.dev/ref/mod#go-install) in line 31?
See [Deprecation of 'go get' for installing executables](https://go.dev/doc/go-get-install-deprecation).

Reply:
+1

Reply:
Wait, sorry, maybe a better question is: do Bank of Anthos Action Runners need Go for anything?
I think in `microservices-demo`, we just need Go for running Go unit tests (for `shippingservice` and `productcatalogservice`)?

Reply:
Fixed.

```
~ go install github.com/google/addlicense@latest
go: downloading github.com/google/addlicense v1.0.0
go: downloading github.com/bmatcuk/doublestar/v4 v4.0.2
go: downloading golang.org/x/sync v0.0.0-20190911185100-cd5d95a43a6e
```